Add support for Circl signature schemes

bwesterb opened the pull request.

bwesterb committed:
Add support for Circl's signature schemes

bwesterb commented:
Updated @armfazh changes in the Circl API

claucece commented:
Looking nice! Just an small comment, I don't think we should add the`certdb/testdb/certstore_development.db`  file...

bwesterb commented:
> Looking nice! Just an small comment, I don't think we should add the`certdb/testdb/certstore_development.db` file...

Oops. Corrected. I've also removed `sqlite_test.db`.
